roles/pulp/templates/settings.py.j2

```diff
@@ -17,3 +17,4 @@ DATABASES = {
 AUTHENTICATION_BACKENDS=['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']
 REMOTE_USER_ENVIRON_NAME="HTTP_REMOTE_USER"
 REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES=('rest_framework.authentication.SessionAuthentication', 'pulpcore.app.authentication.PulpRemoteUserAuthentication')
+ENABLE_ANALYTICS = {{ pulp_enable_analytics | default(false) }}
```

Comment:
Without looking too closely, I think this ends up as `true` or `false` but Python uses `True` and `False`

Reply:
I've updated it to explicitly cast the value as a boolean using `| bool`, ensuring it correctly translates to `True` or `False`.